src/components/BlockchainList/ThreadPart.tsx

```diff
@@ -115,14 +115,19 @@ export default function ({
   return (
     <div className={wrapper}>
       <Replies
+        replyToId={postId}
+        threadId={threadId}
         count={comments.length}
         placeholder={`Reply to ${truncateMiddleIfNeeded(replyingTo, 12)}`}
       />
       {comments.map(
-        ({ timestamp, content, replier, repliedTo, replies }, index) =>
+        ({ timestamp, content, replier, repliedTo, replies, id }, index) =>
           limitThread ? (
             index < limitThread && (
               <CommentWithReplies
+                threadId={threadId}
+                id={id}
```

Comment:
Hm, if `threadId` and `id` are the same things, why do we need both props?

Reply:
No, they are different)